pallets/did/src/errors.rs

```diff
@@ -22,54 +22,54 @@ use crate::did_details::DidVerificationKeyRelationship;
 
 /// All the errors that can be generated when validating a DID operation.
 #[derive(Debug, Eq, PartialEq, TypeInfo)]
-pub enum DidError {
+pub enum Error {
 	/// See [StorageError].
-	StorageError(StorageError),
+	Storage(Storage),
 	/// See [SignatureError].
-	SignatureError(SignatureError),
+	Signature(Signature),
 	/// See [InputError].
-	InputError(InputError),
+	Input(Input),
 	/// An error that is not supposed to take place, yet it happened.
-	InternalError,
+	Internal,
 }
 
-impl From<StorageError> for DidError {
-	fn from(err: StorageError) -> Self {
-		DidError::StorageError(err)
+impl From<Storage> for Error {
+	fn from(err: Storage) -> Self {
+		Error::Storage(err)
 	}
 }
 
-impl From<InputError> for DidError {
-	fn from(err: InputError) -> Self {
-		DidError::InputError(err)
+impl From<Input> for Error {
+	fn from(err: Input) -> Self {
+		Error::Input(err)
 	}
 }
 
 /// Error involving the pallet's storage.
 #[derive(Debug, Eq, PartialEq, TypeInfo)]
-pub enum StorageError {
+pub enum Storage {
 	/// The DID being created is already present on chain.
-	DidAlreadyPresent,
+	AlreadyExists,
 	/// The expected DID cannot be found on chain.
-	DidNotPresent,
+	NotFound,
 	/// The given DID does not contain the right key to verify the signature
 	/// of a DID operation.
-	DidKeyNotPresent(DidVerificationKeyRelationship),
+	DidKeyNotFound(DidVerificationKeyRelationship),
 	/// At least one key referenced is not stored under the given DID.
-	KeyNotPresent,
+	KeyNotFound,
 	/// The maximum number of public keys for this DID key identifier has
 	/// been reached.
-	MaxPublicKeysPerDidExceeded,
+	MaxPublicKeysExceeded,
 	/// The maximum number of key agreements has been reached for the DID
 	/// subject.
 	MaxTotalKeyAgreementKeysExceeded,
 	/// The DID has already been previously deleted.
-	DidAlreadyDeleted,
+	AlreadyDeleted,
 }
 
 /// Error generated when validating a DID operation.
 #[derive(Debug, Eq, PartialEq, TypeInfo)]
-pub enum SignatureError {
+pub enum Signature {
 	/// The signature is not in the format the verification key expects.
 	InvalidSignatureFormat,
```

Comment:
Since we know it's a signature error, we don't require the Signature in the error name? Same with `InvalidSignature` even though we might want to explicitly say that it's the signature part that is wrong and not the nonce part (we also have `InvalidNonce`).

```suggestion
	InvalidFormat,
```

Reply:
I removed the Signature part from the variants and changed from InvalidSignature case to InvalidData